installer/src/test/java/com/teamscale/profiler/installer/LinuxInstallerTest.java

```diff
@@ -0,0 +1,181 @@
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+import static com.teamscale.profiler.installer.utils.UninstallErrorReporterAssert.assertThat;
+import static java.nio.file.attribute.PosixFilePermission.OTHERS_READ;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@EnabledOnOs(OS.LINUX)
+class LinuxInstallerTest {
+
+	private static final int TEAMSCALE_PORT = 8059;
+	private static final String FILE_TO_INSTALL_CONTENT = "install-me";
+	private static final String NESTED_FILE_CONTENT = "nested-file";
+	private static final String ENVIRONMENT_CONTENT = "#this is /etc/environment\nPATH=/usr/bin";
+	private static final String TEAMSCALE_URL = "http://localhost:" + TEAMSCALE_PORT + "/";
+
+
+	private Path sourceDirectory;
+	private Path targetDirectory;
+	private Path etcDirectory;
+
+	private Path installedFile;
+	private Path installedTeamscaleProperties;
+	private Path installedAgentLibrary;
+
+	private Path environmentFile;
+	private Path systemdDirectory;
+	private Path systemdConfig;
+
+	private static MockTeamscale mockTeamscale;
+
+	@BeforeEach
+	void setUpSourceDirectory() throws IOException {
+		sourceDirectory = Files.createTempDirectory("InstallerTest-source");
+		targetDirectory = Files.createTempDirectory("InstallerTest-target").resolve("profiler");
+		etcDirectory = Files.createTempDirectory("InstallerTest-etc");
+
+		environmentFile = etcDirectory.resolve("environment");
+		Files.writeString(environmentFile, ENVIRONMENT_CONTENT);
+
+		systemdDirectory = etcDirectory.resolve("systemd");
+		Files.createDirectory(systemdDirectory);
+		systemdConfig = systemdDirectory.resolve("system.conf.d/teamscale-java-profiler.conf");
+
+		Path fileToInstall = sourceDirectory.resolve("install-me.txt");
+		Files.writeString(fileToInstall, FILE_TO_INSTALL_CONTENT, StandardOpenOption.CREATE);
+
+		Path nestedFileToInstall = sourceDirectory.resolve("lib/teamscale-jacoco-agent.jar");
+		Files.createDirectories(nestedFileToInstall.getParent());
+		Files.writeString(nestedFileToInstall, NESTED_FILE_CONTENT, StandardOpenOption.CREATE);
+
+		installedFile = targetDirectory.resolve(sourceDirectory.relativize(fileToInstall));
+		installedTeamscaleProperties = targetDirectory.resolve("teamscale.properties");
+		installedAgentLibrary = targetDirectory.resolve("lib/teamscale-jacoco-agent.jar");
+	}
+
+	@BeforeAll
+	static void startFakeTeamscale() {
+		mockTeamscale = new MockTeamscale(TEAMSCALE_PORT);
+	}
+
+	@AfterAll
+	static void stopFakeTeamscale() {
+		mockTeamscale.shutdown();
+	}
+
+	@Test
+	void successfulInstallation() throws FatalInstallerError, IOException {
+		install();
+
+		assertThat(Files.getPosixFilePermissions(installedTeamscaleProperties)).contains(OTHERS_READ);
+		assertThat(Files.getPosixFilePermissions(installedFile)).contains(OTHERS_READ);
+
+		assertThat(environmentFile).content().isEqualTo(ENVIRONMENT_CONTENT
+														+ "\nJAVA_TOOL_OPTIONS=-javaagent:" + installedAgentLibrary
```

Comment:
The formatting looks a bit odd here 🤔

Reply:
I think the IDE is set to align the strings with the `(`. Autoformat puts the strings there automatically. Should we change that setting?

Reply:
Ah it looked especially bad for me because my GitHub setting was tab=8 spaces. With 4 spaces it looks okish, but if we can easily change this I would tend to do so. WDTY?